src/Engine/ProtoCore/ExtendedLibraries/FunctionObject.ds

```diff
@@ -479,33 +479,7 @@ def SortByFunction(list : var[]..[], func:_FunctionObject)
         {
             _keys[_index] = __Apply(func, list[_index]);
         }
-        return = Sorting.sortByKey(list, _keys)[0];
```

Comment:
another concern I have is this file:
https://github.com/DynamoDS/Dynamo/blob/11f90d72e1047fcea9ece3818e90cf420890864d/src/DynamoCore/BuiltInAndOperators/BuiltIn.Migrations.xml

just a question for you: does this migration work correctly now that sortbykey and group by key are zero touch nodes?